parm/postcntrl_gefs_chem.xml

```diff
@@ -224,6 +224,12 @@
        <scale>9.0</scale>
     </param>
 
+    <param>
+       <shortname>DUST10_SFC_MASS_CON</shortname>
+       <table_info>NCEP</table_info>
+       <scale> 9.0</scale>
```

Comment:
Maybe there is no space for <scale> 9.0</scale>

Reply:
delete the space